templates/basic_answerfile.txt.j2

```diff
@@ -0,0 +1,111 @@
+OVESETUP_CONFIG/firewallChangesReview=none:None
+OVESETUP_CONFIG/remoteEngineHostSshPort=none:None
+OVESETUP_CONFIG/fqdn=str:{{ ansible_fqdn }}
+OVESETUP_CONFIG/storageType=none:None
+{% if ovirt_engine_setup_require_rollback is defined %}
+OSETUP_RPMDISTRO/requireRollback=bool:{{ovirt_engine_setup_require_rollback}}
+{% else %}
+OSETUP_RPMDISTRO/requireRollback=none:None
+{% endif %}
+OSETUP_RPMDISTRO/enableUpgrade=bool:True
+OVESETUP_DB/secured=bool:False
+OVESETUP_DB/host=str:{{ovirt_engine_setup_db_host}}
+OVESETUP_DB/user=str:{{ovirt_engine_setup_db_user}}
+{% if ovirt_engine_setup_db_password is defined %}
+OVESETUP_DB/password=str:{{ovirt_engine_setup_db_password}}
+{% else %}
+OVESETUP_DB/password=none:None
+{% endif %}
+OVESETUP_DB/dumper=str:pg_custom
+OVESETUP_DB/database=str:{{ovirt_engine_setup_db_name}}
+OVESETUP_DB/fixDbViolations=none:None
+OVESETUP_DB/port=int:{{ovirt_engine_setup_db_port}}
+OVESETUP_DB/filter=none:None
+OVESETUP_DB/restoreJobs=int:2
+OVESETUP_DB/securedHostValidation=bool:False
+{% if ovirt_engine_setup_dwh_db_configure is defined and ovirt_engine_setup_dwh_db_configure %}
+OVESETUP_DWH_DB/secured=bool:False
+OVESETUP_DWH_DB/host=str:{{ovirt_engine_setup_dwh_db_host}}
+OVESETUP_DWH_DB/user=str:{{ovirt_engine_setup_dwh_db_user}}
+OVESETUP_DWH_DB/database=str:{{ovirt_engine_setup_dwh_db_name}}
+OVESETUP_DWH_DB/port=int:{{ovirt_engine_setup_dwh_db_port}}
+{% else %}
+OVESETUP_DWH_DB/secured=none:None
+OVESETUP_DWH_DB/host=none:None
+OVESETUP_DWH_DB/user=none:None
+OVESETUP_DWH_DB/password=none:None
+OVESETUP_DWH_DB/database=none:None
+OVESETUP_DWH_DB/port=none:None
+{% endif %}
+{% if ovirt_engine_setup_dwh_db_password is defined %}
+OVESETUP_DWH_DB/password=str:{{ovirt_engine_setup_dwh_db_password}}
+{% else %}
+OVESETUP_DWH_DB/password=none:None
+{% endif %}
+OVESETUP_DWH_DB/dumper=str:pg_custom
+OVESETUP_DWH_DB/filter=none:None
+OVESETUP_DWH_DB/restoreJobs=int:2
+{% if ovirt_engine_setup_dwh_db_configure is defined and ovirt_engine_setup_dwh_db_configure %}
+OVESETUP_DWH_DB/securedHostValidation=bool:False
+{% else %}
+OVESETUP_DWH_DB/securedHostValidation=none:None
+{% endif %}
+OVESETUP_ENGINE_CORE/enable=bool:True
+OVESETUP_CORE/engineStop=none:None
+OVESETUP_SYSTEM/memCheckEnabled=bool:False
+OVESETUP_SYSTEM/nfsConfigEnabled=bool:False
+OVESETUP_PKI/organization=str:{{ ovirt_engine_setup_organization }}
+OVESETUP_PKI/renew=none:None
+{% if ovirt_engine_setup_configure_iso_domain is defined %}
+OVESETUP_CONFIG/isoDomainName=str:{{ ovirt_engine_setup_iso_domain_name }}
+OVESETUP_CONFIG/isoDomainACL=str:{{ ovirt_engine_setup_iso_domain_acl }}
+OVESETUP_CONFIG/isoDomainMountPoint=str:{{ ovirt_engine_setup_iso_domain_path }}
+OVESETUP_SYSTEM/nfsConfigEnabled=bool:True
+{% else %}
+OVESETUP_CONFIG/isoDomainName=none:None
+OVESETUP_CONFIG/isoDomainACL=none:None
+OVESETUP_CONFIG/isoDomainMountPoint=none:None
+OVESETUP_SYSTEM/nfsConfigEnabled=bool:False
+{% endif %}
+OVESETUP_CONFIG/engineHeapMax=str:1024M
```

Comment:
Please remove engineHeapMin and engineHeapMax , so that engine-setup does its guesswork. See also: https://bugzilla.redhat.com/1185411

Reply:
Done